src/test/CMakeLists.txt

```diff
@@ -40,7 +38,11 @@ add_subdirectory(pybind)
 if(${WITH_RADOSGW})
   add_subdirectory(rgw)
 endif(${WITH_RADOSGW})
-add_subdirectory(rbd_mirror)
+if(WITH_RBD)
```

Comment:
@tchaikov 
I tried to include all under 1 if test.
And the order within the exclusion is maintained.